Documentation/Index.rst

```diff
@@ -92,18 +93,14 @@ Install DFG-Viewer and Kitodo.Presentation via Composer
 
 Composer commands::
 
-    # make sure you haven't set the platform php version to 7.2
-    composer config platform.php 7.3
-    # install DFG-Viewer extension
-    composer require slub/dfgviewer:^6.0
+    composer require slub/dfgviewer:^6.1
 
-This will install the DFG-Viewer 6.0 extension and Kitodo.Presentation 4.0 from
+This will install the DFG-Viewer 6.1 extension and Kitodo.Presentation 5.0 from
 `Packagist <https://github.com/slub/dfg-viewer>`_.
 
-Install the Extension via extension manager or CLI::
+Configure the extension via cli::
 
-    ./vendor/bin/typo3 extension:activate dlf
-    ./vendor/bin/typo3 extension:activate dfgviewer
+    ./vendor/bin/typo3 install:extensionsetupifpossible
```

Comment:
This does not work with TYPO3 12:
```
There are no commands defined in the "install" namespace.
```

Reply:
That is part of the `helhum/typo3-console` not TYPO3 which is responsible for the legacy commands.

Reply:
Did I miss something when I installed TYPO3 and the extensions for DFG Viewer? Obviously I did not get helhum/typo3-console.